editor.planx.uk/src/pages/FlowEditor/components/Sidebar/PublishDialog.tsx

```diff
@@ -250,13 +268,34 @@ export const ValidationChecks = (props: {
 
   return (
     <Box pb={2}>
-      <Typography variant="h4" component="h3">
+      <Typography
+        variant="h4"
+        component="h3"
+        gutterBottom
+        sx={{ display: "flex", alignItems: "center" }}
+      >
         Validation checks
+        <LightTooltip
+          title="Validation checks are automatic tests that scan your service and highlight when content changes introduce an error, like incorrectly using a component type or breaking an integration."
+          placement="right"
+        >
+          <IconButton>
+            <Help color="primary" />
+          </IconButton>
+        </LightTooltip>
       </Typography>
-      <List sx={{ pb: 2 }}>
+      <ValidationSummary validationChecks={validationChecks} />
+      <List>
         {validationChecks.map((check, i) => (
-          <ListItem key={i} disablePadding>
-            <ListItemIcon sx={{ minWidth: (theme) => theme.spacing(3) }}>
+          <ListItem
+            key={i}
+            dense
+            sx={{
+              backgroundColor: (theme) => theme.palette.background.default,
+              border: (theme) => `1px solid ${theme.palette.border.light}`,
+            }}
```

Comment:
Small style nit, `marginTop: "-1px"` will eliminate the double border between each `<ListItem>`

Reply:
Now updated, thank you for spotting!